CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-# Upcoming release
+# 9.1.0
```

Comment:
Since we're re-breaking the .git handling, maybe 10.0.0?

Reply:
It's probably the cleanest thing to do. I wouldn't mind going to 10.0.0.

Reply:
done